src/vm/docs/mod.rs

```diff
@@ -1180,6 +1180,19 @@ one of the following error codes:
 "
 };
 
+const STX_GET_BALANCE: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(stx-get-balance owner)",
+    description: "`stx-get-balance` is used to query the STX balance of the `owner` principal.
+
+This function returns the STX balance of the `owner` principal. In the event
+that the `owner` principal doesn't exist, it returns 0.
```

Comment:
In SIP 005, we say that all principals exist (i.e. in the same way that all numbers exist), but that not all principals are materialized.  What do you think of the text `In the event that the owner principal isn't materialized, it returns 0` as an alternative?